### PR TITLE
Ruby and MATLAB syntax support

### PR DESCRIPTION
**Pull request by dig386**
This commit adds support of Ruby and MATLAB/Octave syntax highlighting to WCM Commander

### CODE REVIEW

File: install-files/share/wcm/shl/config.cfg [hunk 1]
```diff
@@ -27,7 +29,7 @@ rule first ("#!*/python*")		"python";
 rule mask ("*\\shl\\config.shl","*/shl/config.shl")	"wcm-shl-main-v1";
 rule mask ("*\\shl\\config.cfg","*/shl/config.cfg")	"wcm-shl-main";
 rule mask ("*\\shl\\*.shl","*/shl/*.shl")		"wcm-shl";
-rule mask ("*.c","*.cc","*.cpp","*.h","*.hpp","*.C","*.m")	"C++";
```
Comment (corporateshark) [inline install-files/share/wcm/shl/config.cfg]: That was Objective-C

Comment (dig386) [inline install-files/share/wcm/shl/config.cfg]: Both Objective-C and MATLAB (or GNU Octave) use files with .m extension (before your comment I even didn't know that it was for Objective-C and thought that it is an attempt to add some elements of syntax highlighting to .m scripts). But if Objective-C is considered more actual is it possible to just include matlab.shl and comment rules for MATLAB?

Comment (corporateshark) [inline install-files/share/wcm/shl/config.cfg]: Yeah, sure. Just roll back lines changing ``.m`` and I will merge this pull request.

**Comment by dig386** [comment]
I've modified config.cfg. Now the default variant is for Objective-C, but a user can read comments and switch to MATLAB.

**Pull request merged**